code/script.js

```diff
@@ -1,47 +1,317 @@
+const ambulanceAnswer = (message) => {
+   if (questionNumber === 1) {
+       userReply(message);
+       setTimeout(() => ambulanceCase(message), 500);
+   } else if (questionNumber === 2) {
+       userReply(message);
+       setTimeout(() => confirmAmbulance(message),500 ) 
+       restart();
+   } 
+}
+
+//If user choose minor injury - the answer will return to the message here
+const injuryAnswer = (message) => {
+    if (questionNumber === 1) {
+        userReply(message);
+        setTimeout(() => bookingDoctor(message), 1000); // quesion 2
+    } else if (questionNumber === 2) {
+        userReply(message)
+        setTimeout(() => bookingTime(message), 1000);
+    } else if (questionNumber === 3) {
+        userReply(message);
+        setTimeout(() => doctorOption(message), 1000);
+    } else if (questionNumber === 4) {
+        setTimeout(() => userReply(message), 300);
+        setTimeout(() => confirmDoctor(message), 1000);
+        setTimeout(restart, 1500);
+    }
+}
+
+
+
+///////////////////////////// QUESTION 1 FOR BOTH CASES - START HERE
+//Question 1 - Default Greeting - Show 2 different cases
+const greeting = (message) => {
+    questionNumber = 1;
+    botReply(`Hi, I am Dr.Strange, what can I help with today?`);
+    showInjuryCase();
+  } 
+
+//This function is to be added in question 1 - Show 2 different cases
+function showInjuryCase(message) {
+
+    //Show two button
+    inputWrapper.innerHTML = `
+    <button type="button" id="ambulance"> 🚑 In need of an ambulance</button>
+    <button type="button" id="minor-injury"> 🤕 Minor Injury </button>
+  `; 
+  
+    //Add event listener for "in need of ambulance" button
+     const ambulanceBtn = document.getElementById("ambulance");
+     ambulanceBtn.addEventListener("click", () => {
+      ambulanceAnswer("In need of an ambulance");
+     pop.play();
+   });
+  
+     //Add event lisner for "minor injury" case
+     const injuryBtn = document.getElementById("minor-injury");
+     injuryBtn.addEventListener("click", ()=> {
+     
+     //injuryAnswer
+     injuryAnswer("Minor Injury");
+     pop.play();
+  });
+  }
+
+
+///////////////////////// IN NEED OF AMBULANCE
+
+// Question 2 - Ambulance - Request address from user
+ function ambulanceCase (message) {
+    questionNumber++;
+   
+       setTimeout(() => botReply(`oh, you are ${message} 🥺`),300);
+       setTimeout(() => botReply(`Where can we pick you up? 🤔🏡`), 800);
+
+      //Change the buttons to address input
+      inputWrapper.innerHTML = `
+      <form id="address-form">
+      <input id="address-input" type="text" />
+      <button class="send-btn" type="submit"> Send </button>
+      </form>`;
+
+      //Create variables for the input form and input field
+      const addressInput = document.querySelector("#address-input");
+      const addressForm = document.getElementById("address-form");
+
+      //Set condition for address input
```

Comment:
You have very good comments in all the functions but we're not sure what this comment means? 😅  🙈